mitiq/interface/mitiq_cirq/cirq_utils.py

```diff
@@ -32,6 +33,37 @@
 ]
 
 
+# Executors.
+def sample_bitstrings(
+    circuit: cirq.Circuit,
+    noise_model: cirq.NOISE_MODEL_LIKE = cirq.amplitude_damp,  # type: ignore
+    noise_level: Tuple[float] = (0.01,),
```

Comment:
Something like `cirq.asymmetric_depolarize` has three arguments - `p_x`, `p_y`, and `p_z`. `noise_level` as a `Tuple[float]` instead of just a `float` is to allow for this.